### DISCUSSION TITLE
XP Pen Deco 01 V2 Area Wrong Even After Troubleshooting

### DISCUSSION THREAD

**Discussion by kujotakezo** [OpenTabletDriver/OpenTabletDriver#2727] [Q&A] [answered]
Hi,

As other XP Pen Deco 01 V2 users have commented, the area for my tablet is incorrect - for me even when selecting a full area only around a quarter of my tablet in the bottom right is detected (for others it was the top left but it's a similar issue). A solution was given on these posts to extract and import a .json file which should fix the issue (#1571), but this didn't work for me - my area is still constrained to just the bottom right corner. I have also done other basic troubleshooting steps such as uninstalling XP-Pen drivers which didn't worth either. 

Just to note, I didn't have this issue with native XP Pen drivers - full area worked just fine there but Opentablet has much better area adjustability so I would like to use it if possible. I have attached a diagnostics file.

[xppen diagnostics.txt](https://github.com/OpenTabletDriver/OpenTabletDriver/files/11398483/xppen.diagnostics.txt)

**Comment by gonX** [comment]
Is this still the case in the latest release version, currently 0.6.3.0?

**Reply by kujotakezo** [reply, marked as answer]
Hi, the issue did resolve with the latest build around that time "0.7.0.0+f7df4de2", though at the time filters were very unstable so I couldn't install any of them. But I just tried the 0.6.3.0 version and it works perfectly! It correctly recognises my tablet as "(variant 2)" so the area issues are fixed, and I can install filters which seem to be working great.
